src/server/rpc/procedures/twitter/twitter.js

```diff
@@ -8,23 +8,17 @@
 // This will use the Twitter API to allow the client to execute certain Twitter functions within NetsBlox
 
 'use strict';
-
-const logger = require('../utils/logger')('twitter');
-const request = require('request');
-const baseURL = 'https://api.twitter.com/1.1/';
+const ApiConsumer = require('../utils/api-consumer');
 var KEY = process.env.TWITTER_BEARER_TOKEN;
 
 // make sure key starts with bearer
-if ( KEY && !KEY.startsWith('Bearer ')) KEY = 'Bearer ' + KEY;
+if (KEY && !KEY.startsWith('Bearer ')) KEY = 'Bearer ' + KEY;
 
-var options = {
-    url: baseURL,
-    headers: { 
-        'Authorization': KEY,
-        'gzip': true
-    },
-    json: true
-};
+const TwitterConsumer = new ApiConsumer('twitter', 'https://api.twitter.com/1.1/', {
+    cache: {
+        ttl: 5 * 60
```

Comment:
this might be long for Twitter, it might affect some projects, what do you think about 30 seconds or no cache at all?